src/identicon.js

```diff
@@ -3,14 +3,19 @@ let byteIndex = 0
 
 function setBytes (hash) {
   bytes = hash
-  byteIndex = hash[hash.length - 1] % hash.length // set the starting point
+  // byteIndex = hash[hash.length - 1] % hash.length // set the starting point
+  byteIndex = 0
 }
 
 // returns a value from 0 to 1, determined by the next byte in the hash
 function value () {
-  const result = bytes[byteIndex] / 256.0
-  byteIndex = (byteIndex + 1) % bytes.length
-  return result
+  const getByte = () => {
+    const result = bytes[byteIndex]
+    byteIndex = (byteIndex + 1) % bytes.length
+    return result
+  }
+
+  return (getByte() + getByte()) / 512 // use 2 bytes per value to make sure we're using all of the hash
```

Comment:
Adding is no good because it means that a hash with bytes `[0, 255, 0, 255 ...]` is equivalent to `[1, 244, 1, 244, ...]`

Reply:
What you could do instead is
```suggestion
  return (getByte() * 256 + getByte())/(256*256) // use 2 bytes per value to make sure we're using all of the hash
```

Reply:
hmm. I considered that, but then I don't think the least significant byte will actually have any effect on the final image, so it's basically equivalent to just using every other byte. Will think about it more.

Reply:
XOR is no good either. It has the same many equivalent pairs problem as addition e.g. `251 ^ 4` = `253 ^ 2` = `254 ^ 1` = `255 ^ 0` = `255`

Reply:
Perhaps it would be possible to take advantage of primes. log<sub>2</sub>(2<sup>A</sup> * 3<sup>B</sup>)

Reply:
that doesn't seem that good either. maybe (2<sup>A</sup> * 3<sup>B</sup>)/(2<sup>256</sup> * 3<sup>256</sup>)

Reply:
Decision was to move on for now and revisit when we need it for production.